CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 
-## v0.12.0-dev
+## v0.12.0
+
+This is the first official Raspberry Pi Zero system release. It was forked
+off `nerves_system_rpi` so previous versions are from that project. These
```

Comment:
Minor point of confusion as I read this sentence: I initially read "These" as referring to this system's project. Maybe change to "Those" or "`nerves_system_rpi` also works..."?